esx_service/cli/vmdkops_admin.py

```diff
@@ -943,55 +895,14 @@ def tenant_vm_ls(args):
     rows = generate_tenant_vm_ls_rows(vms)
     print(cli_table.create(header, rows))
 
-def default_privileges():
-     privileges = [{'datastore': 'datastore1',
-                     'global_visibility': 0,
-                      'create_volume': 0,
-                      'delete_volume': 0,
-                     'mount_volume': 0,
-                      'max_volume_size': 0,
-                      'usage_quota': 0}]
-     return privileges
-
-def set_privileges(rights, privileges, value):
-    """ set or unset privileges based rights passed by command line """
-    if 'create' in rights:
-        privileges[auth_data_const.COL_CREATE_VOLUME] = value
-
-    if 'delete' in rights:
-        privileges[auth_data_const.COL_DELETE_VOLUME] = value
-
-    if 'mount' in rights:
-        privileges[auth_data_const.COL_MOUNT_VOLUME] = value
-
-    if 'all' in rights:
-        privileges[auth_data_const.COL_CREATE_VOLUME] = value
-        privileges[auth_data_const.COL_DELETE_VOLUME] = value
-        privileges[auth_data_const.COL_MOUNT_VOLUME] = value
 
-def generate_privileges(args):
-    """ Generate privileges based on CLI argument """
-    privileges = default_privileges()[0]
-    privileges[auth_data_const.COL_DATASTORE] = args.datastore
-
-    if args.rights:
-        set_privileges(args.rights, privileges, 1)
-
-    if args.volume_maxsize:
-        size_in_MB = convert.convert_to_MB(args.volume_maxsize)
-        privileges[auth_data_const.COL_MAX_VOLUME_SIZE] = size_in_MB
-
-    if args.volume_totalsize:
-        size_in_MB = convert.convert_to_MB(args.volume_totalsize)
-        privileges[auth_data_const.COL_USAGE_QUOTA] = size_in_MB
-
-    return privileges
 
 def tenant_access_add(args):
     """ Handle tenant access command """
     error_info = auth_api._tenant_access_add(name=args.name,
                                              datastore=args.datastore,
-                                             rights=args.rights,
+                                             switch_default_datastore=args.switch_default_datastore,
+                                             allow_create=args.allow_create,
```

Comment:
nit: the rest of params do not say "switch_allow_create", why default_datastore does ?